src/scores/continuous/standard_impl.py

```diff
@@ -373,9 +372,9 @@ def pbias(
         fcst: Forecast or predicted variables.
         obs: Observed variables.
         reduce_dims: Optionally specify which dimensions to reduce when
-            calculating the percentage additive bias. All other dimensions will be preserved.
+            calculating the percent bias. All other dimensions will be preserved.
         preserve_dims: Optionally specify which dimensions to preserve when
-            calculating the additive bias percentage. All other dimensions will be reduced. As a
+            calculating the percent bias. All other dimensions will be reduced. As a
```

Comment:
I changed "the additive bias percentage" to "the percent bias".

I wasn't sure whether "the" should be left in. If "the" should be deleted, please let me know.